server/src/test/java/com/ca/lsp/core/cobol/engine/CobolLanguageEngineTest.java

```diff
@@ -123,11 +127,68 @@ void testLanguageEngineRun() {
     ResultWithErrors<SemanticContext> expected =
         new ResultWithErrors<>(
             new SemanticContext(
-                Map.of(), Map.of(), Map.of(), Map.of(), Map.of(), Map.of(), expectedOutlineTree),
+                Map.of(),
+                Map.of(),
+                Map.of(),
+                Map.of(),
+                getConstantDefinitions(),
+                Map.of(),
+                Map.of(),
+                Map.of(),
+                expectedOutlineTree),
             List.of(error));
 
     ResultWithErrors<SemanticContext> actual = engine.run(URI, TEXT, PROCESSING_MODE);
 
     assertEquals(expected, actual);
   }
+
+  private Map<String, Collection<Location>> getConstantDefinitions() {
+    Location mockLocation = new Location();
+    Multimap<String, Location> result = HashMultimap.create();
+    result.put("EIBAID", mockLocation);
```

Comment:
The third time you describe constants. It will be good to define them once.

Reply:
I agree. I am planning to split this test and leave only the significant logic as a part of reducing the level of duplication in the entire test set.